tests/cli/outputs/test_storage_layout.py

```diff
@@ -34,5 +34,5 @@ def public_bar():
             "slot": 3,
         },
         "baz": {"type": "Bytes[65]", "location": "storage", "slot": 4},
-        "bar": {"type": "uint256", "location": "storage", "slot": 9},
+        "bar": {"type": "uint256", "location": "storage", "slot": 8},
```

Comment:
What was the change here?

Reply:
Allocator always used two words of overhead for bytearrays. This was because in certain situations in the calling convention, bytearrays were interpreted in-place and so they needed an extra word allocated for the offset word. (The ABI encoding for a bytestring is basically `0x20 <offset in the dynamic section> | <1 length word> | <bytestring data>`). In this PR, there is no implicit interpretation of bytearrays in this way, and so the allocator does not need to allocate the extra word anymore.

Reply:
You can calculate on pen/paper - `baz` required 2 + 1 + 1 words before, and now only requires 2 + 1 words.